packages/cli/workspace-loader/src/types/Workspace.ts

```diff
@@ -36,7 +36,9 @@ export interface Spec {
 
 export interface SpecImportSettings {
     audiences: string[];
+    sdkLanguage: generatorsYml.GenerationLanguage | undefined;
```

Comment:
why do we add the sdk language here ? we should just pass it from the generator invocation